libdnf/base/goal.cpp

```diff
@@ -946,7 +947,12 @@ base::Transaction Goal::resolve(bool allow_erasing) {
     }
     ret |= p_impl->rpm_goal.resolve();
 
+    if (cfg_main.debug_solver().get_value()) {
+        p_impl->rpm_goal.write_debugdata(std::filesystem::path(cfg_main.debugdir().get_value()) / "packages");
+    }
```

Comment:
Please can you add into event log that debug data were saved into into <dir> after success of `p_impl->rpm_goal.write_debugdata(std::filesystem::path(cfg_main.debugdir().get_value()) / "packages");`